### ISSUE TITLE
Bower?

### ISSUE THREAD

**Issue by IngwiePhoenix** [ace-subido/css3-microsoft-metro-buttons#43]
Is there a possibility, that this plugin will be published to Bower at some time?

**Comment by martam9** [comment]
mmm...is it a joke or you want to post me?

**Comment by IngwiePhoenix** [comment]
Nope, I am not kidding. I am using WebPack for my application and you can import bower modules into it as well. It would be handy to be able to pull  this nice Metro loo kinto said package easier. :)

**Comment by martam9** [comment]
![image](https://cloud.githubusercontent.com/assets/1003962/7334373/7badadca-eb91-11e4-814f-460f5075eecf.png)
![image](https://cloud.githubusercontent.com/assets/1003962/7334375/7eedd47e-eb91-11e4-9c8a-b3534ca8e1ca.png)
![Uploading image.png . . .]()
![Uploading image.png . . .]()

**Comment by martam9** [comment]
i sometimes don t want the full screen it s too big

**Comment by jack71** [comment]
:+1:

**Comment by martam9** [comment]
lol

2015-06-29 13:52 GMT+02:00 jack71 <notifications@github.com>:

> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ace-subido/css3-microsoft-metro-buttons/issues/43#issuecomment-116626891>
> .
>



-- 
MM

**Comment by martam9** [comment]
come faccio a vedere un mp4 devo mettere dvx?

-- 
MM

**Comment by martam9** [comment]
i need to know how to open a file sent with google drive
i m downloading dvx

2015-06-30 12:11 GMT+02:00 Marta Marras <marta.marras9@gmail.com>:

> lol
>
> 2015-06-29 13:52 GMT+02:00 jack71 <notifications@github.com>:
>
>> [image: :+1:]
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ace-subido/css3-microsoft-metro-buttons/issues/43#issuecomment-116626891>
>> .
>>
>
>
>
> --
> MM
>



-- 
MM

**Comment by roy128** [comment]
:+1:
